Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/presencetracing/organizer/create/TraceLocationCreateViewModel.kt

```diff
@@ -111,7 +111,7 @@ class TraceLocationCreateViewModel @AssistedInject constructor(
         )
     }
 
-    private fun String.isTextFormattedCorrectly() = trim().length in 1..100 && !contains('\n')
+    private fun String.isTextFormattedCorrectly() = trim().length in 1..MAX_LENGTH_DESCRIPTION && !contains('\n')
```

Comment:
if we have two different constants, then we need to consider them both while validations. this function is used to validate `description` and `address`. if both constants are not the same the other will fail